index.html

```diff
@@ -0,0 +1,166 @@
+                            <i class="fab fa-facebook"></i>
+                        </a>
+                        <a href="https://www.linkedin.com/in/rostyslav-l-815b271b0/" class="social-media-link">
+                            <i class="fab fa-linkedin-in"></i>
+                        </a>
+                    </div>
+                </section>
+            </div>
+            <div class="container-right"> 
+            <main>
+                    <!-- Education Section -->
+                        <section class="section">
+                            <h2 class="education-heading "> <i class="fas fa-user-graduate "></i>  Education </h2>
+                            <hr />
+                                <div>
+                                    <h2><i class="fas fa-university"></i>   Gdansk University of Technology, Gdansk </h2>
+                                    <details><summary> <strong> Master Degree </strong>   </summary>       
+                                        <ul class="study-skills-text">
+                                            <li> <h5> <b>Field of Study </b> :</h5> Electronics and Telecommunications       </li>
+                                            <li> <h5> <b>Scope of Study </b> :</h5> Teleinformation Networks and Systems     </li>
+                                            <li> <h5> <b>Discipline     </b> :</h5> Information and Communication Technology </li>
+                                            <li> <h5> <b>Terms of study </b> :</h5> 28.02.2017 - 19.11.2019                  </li>
+                                        </ul>   
+                                    </details> 
+                                    <h2><i class="fas fa-university"></i>  COLLEGE OF COMMUNICATIONS ,KYIV </h2> 
+                                    <details><summary><strong>  Engineer Degree </strong> </summary> 
+                                        <ul class="study-skills-text">
+                                            <li> <h5> <b> Field of study </b>: </h5> Telecommunications                       </li>
+                                            <li> <h5> <b> Scope of Study </b>: </h5> Teleinformation Networks and Systems     </li>
+                                            <li> <h5> <b> Discipline     </b>: </h5> Information and Communication Technology </li>
+                                            <li> <h5> <b> Terms of study </b>: </h5> 01.09.2010 - 01.02.2017                  </li>
+                                        </ul>      
+                                    </details>     
+                                </div>
+                        </section>
+                        <!-- Skills Section -->
+                        <section class="skills-section">
+                            <h1 > <i class="fas fa-tools"></i>   Skills </h1>
+                            <hr />
+                            <div>  
+                                <div>
```

Comment:
here you probably don't need to wrap each skill inside `div`. Also it looks more like enumeration/list